creating-virtual-machines/dedicated-cpu.adoc

```diff
@@ -150,6 +150,9 @@ data:
 Sidecar containers and CPU allocation overhead
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+*Note:* In order to run sidecar containers, KubeVirt requires the `Sidecar` feature gate to be enabled by adding `Sidecar`
+to the kubevirt-config ConfigMap's feature-gates field.
```

Comment:
nitpicking, but can you put `kubevirt-config` and `feature-gates` into backticks?

Reply:
oh, I missed that comment, sure.

Reply:
done

Reply:
thanks!